plugins/uefi/fu-uefi-device.c

```diff
@@ -324,6 +324,9 @@ fu_uefi_device_fixup_firmware (FuDevice *device, GBytes *fw, GError **error)
 	} else if (fu_uefi_device_get_kind (self) == FU_UEFI_DEVICE_KIND_FMP) {
 		g_debug ("performing FMP update");
 		return g_bytes_new_from_bytes (fw, 0, fw_length);
+	} else if (fu_uefi_device_get_kind (self) == FU_UEFI_DEVICE_KIND_DELL_TPM_FIRMWARE) {
```

Comment:
I'd rather have an explicit property, e.g. `fu_uefi_device_get_requires_header()` or something like that which could explicitly be unset by FMP and the TPM hardware at device creation time.

Reply:
adjusted